netbox/dcim/filtersets.py

```diff
@@ -1100,6 +1100,10 @@ class DeviceFilterSet(
         queryset=IPAddress.objects.all(),
         label=_('OOB IP (ID)'),
     )
+    has_virtual_device_contexts = django_filters.BooleanFilter(
```

Comment:
The filter name should be singular, as we're not checking for the presence of _multiple_ VDCs. (The corresponding method should be renamed too.)
```suggestion
    has_virtual_device_context = django_filters.BooleanFilter(
```